### PR TITLE
steelMemberShearPanel - Types for Steel Design

### PR DESCRIPTION
**Pull request by dogukankaratas**
- steelMemberShearPanel class has added.
- Enumeration items created. (Some fixes has done in enumeration -there were identical classes, they're removed.)
- Unit test has added.

### CODE REVIEW

File: UnitTests/test_steelMemberShearPanel.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import sys
+import os
+import pytest
+PROJECT_ROOT = os.path.abspath(os.path.join(
+                  os.path.dirname(__file__),
+                  os.pardir)
+)
+sys.path.append(PROJECT_ROOT)
+
+from RFEM.enums import *
```
Comment (OndraMichal) [inline UnitTests/test_steelMemberShearPanel.py:10]: Please specify the imported enum. I think it should be just one.

File: RFEM/TypesForSteelDesign/steelMemberShearPanel.py [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+                user_defined_name: list = [False],
+                definition_type = SteelMemberShearPanelDefinitionType.DEFINITION_TYPE_TRAPEZOIDAL_SHEETING,
+                members: str = "",
+                member_sets: str = "",
+                categories = [SteelMemberShearPanelPositionOnSection.POSITION_ON_UPPER_FLANGE, "FI (+) 35/207 - 0.63 (b: 1) | DIN 18807 | Fischer Profil", SteelMemberShearPanelFasteningArrangement.FASTENING_ARRANGEMENT_EVERY_RIB],
+                parameters = [1, 2, 0.000247, 0.01043],
+                comment: str = '',
+                params: dict = {}):
+        """
+        Args:
```
Comment (OndraMichal) [inline RFEM/TypesForSteelDesign/steelMemberShearPanel.py:17]: I think the argument section could be shorter, not specifying every case. Or at least specify first all arguments and then add any informatio necessary for understanding. I think we don't have to copy whole logic into dosctring. If user wants to know the logic, he ca nread it from the body of the method. 
What do you think?

File: RFEM/TypesForSteelDesign/steelMemberShearPanel.py [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    def __init__(self,
+                no: int = 1,
+                user_defined_name: list = [False],
+                definition_type = SteelMemberShearPanelDefinitionType.DEFINITION_TYPE_TRAPEZOIDAL_SHEETING,
+                members: str = "",
+                member_sets: str = "",
+                categories = [SteelMemberShearPanelPositionOnSection.POSITION_ON_UPPER_FLANGE, "FI (+) 35/207 - 0.63 (b: 1) | DIN 18807 | Fischer Profil", SteelMemberShearPanelFasteningArrangement.FASTENING_ARRANGEMENT_EVERY_RIB],
+                parameters = [1, 2, 0.000247, 0.01043],
+                comment: str = '',
+                params: dict = {}):
```
Comment (OndraMichal) [inline RFEM/TypesForSteelDesign/steelMemberShearPanel.py:15]: default dict value is now None in the main branch. Then add if before params assignment to clentObject.
```
if params:
    for key in params:
        clientObject[key] = params[key] 
```

File: UnitTests/test_steelMemberShearPanel.py [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    shearPanel_1 = Model.clientModel.service.get_steel_member_shear_panel(1)
+    shearPanel_2 = Model.clientModel.service.get_steel_member_shear_panel(2)
+    shearPanel_3 = Model.clientModel.service.get_steel_member_shear_panel(3)
+
+    assert shearPanel_1.no == 1
+    assert shearPanel_2.definition_type == "DEFINITION_TYPE_TRAPEZOIDAL_SHEETING"
+    assert shearPanel_3.panel_length == 2.0
+
+
+    Model.clientModel.service.finish_modification()
```
Comment (OndraMichal) [inline UnitTests/test_steelMemberShearPanel.py:44]: Add empty line at the end :-)

**Review by OndraMichal — CHANGES_REQUESTED** [review]

File: UnitTests/test_steelMemberShearPanel.py (hunk 1 above)
Comment (dogukankaratas) [reply UnitTests/test_steelMemberShearPanel.py:10]: fixed

File: UnitTests/test_steelMemberShearPanel.py (hunk 4 above)
Comment (dogukankaratas) [reply UnitTests/test_steelMemberShearPanel.py:44]: done

File: RFEM/TypesForSteelDesign/steelMemberShearPanel.py (hunk 2 above)
Comment (dogukankaratas) [reply RFEM/TypesForSteelDesign/steelMemberShearPanel.py:17]: Can be done, but didn't want to delete when it's already there :) I will be more specific next time.

**Review by OndraMichal — APPROVED** [review]

**Pull request merged**